packages/paste-icons/tools/templates/reactIconTemplate.js

```diff
@@ -20,7 +20,7 @@ const ${componentName}: React.FC<${componentName}Props> = ({as, display, size, i
   return (
     <IconWrapper as={as} display={display} size={size} iconColor={iconColor}>
       <UID>
-        {uid => (
+        {titleId => (
```

Comment:
New SVGR forces this variable name, so I had to swap `uid` to `titleId` in the upgrade.